lightning/src/ln/msgs.rs

```diff
@@ -2478,6 +2485,17 @@ impl Writeable for OutboundOnionPayload {
 					(6, short_channel_id, required)
 				});
 			},
+			Self::TrampolineEntrypoint {
+				amt_to_forward, outgoing_cltv_value, multipath_trampoline_data: ref payment_data,
```

Comment:
nit:
```suggestion
				amt_to_forward, outgoing_cltv_value, ref multipath_trampoline_data,
```
This is my one complaint about the auto-rename feature, that it isn't smart about this